src/WebJobs.Script.WebHost/Configuration/ScriptApplicationHostOptionsSetup.cs

```diff
@@ -9,22 +9,23 @@
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost.Configuration
 {
-    public class ScriptApplicationHostOptionsSetup : IConfigureNamedOptions<ScriptApplicationHostOptions>
+    public class ScriptApplicationHostOptionsSetup : IConfigureNamedOptions<ScriptApplicationHostOptions>, IDisposable
     {
         public const string SkipPlaceholder = "SkipPlaceholder";
-
+        private readonly IOptionsMonitorCache<ScriptApplicationHostOptions> _cache;
         private readonly IConfiguration _configuration;
-        private readonly IEnvironment _environment;
-
-        public ScriptApplicationHostOptionsSetup(IConfiguration configuration)
-            : this(configuration, SystemEnvironment.Instance)
-        {
-        }
+        private readonly IOptionsMonitor<StandbyOptions> _standbyOptions;
+        private readonly IDisposable _disposable;
```

Comment:
nit: Rename `_disposable` to something more meaningful here (e.g. `_standbyChangeSubscription`)